src/mesh/mesh_io.jl

```diff
@@ -104,6 +104,25 @@ function save_mesh_file(mesh::CurvedMesh, output_directory)
 end
 
 
+function save_mesh_file(mesh::UnstructuredQuadMesh, output_directory)
+  # Create output directory (if it does not exist)
+  mkpath(output_directory)
+
+  filename = joinpath(output_directory, "mesh.h5")
+
+  # Open file (clobber existing content)
+  h5open(filename, "w") do file
+    # Add context information as attributes
+    attributes(file)["mesh_type"] = get_name(mesh)
+    attributes(file)["ndims"] = ndims(mesh)
+    attributes(file)["size"] = length(mesh)
+    attributes(file)["mesh_filename"] = mesh.filename
+  end
+
+  return filename
+end
```

Comment:
This doesn't really save the mesh, does it? Is it just a placeholder for now?

Reply:
Yes, it is similar to the `CurvedMesh`version right above it. The mesh is actually reconstructed within `Trixi2Vtk` based on these attributes given here.

Reply:
Maybe such a comment would be nice here